dev-python/requests-cache/requests-cache-0.5.2.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+PYTHON_REQ_USE="sqlite"
+
+inherit distutils-r1
+
+HOMEPAGE="https://pypi.org/project/requests-cache/"
+DESCRIPTION="Persistent cache for requests library"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+RDEPEND="dev-python/requests[${PYTHON_USEDEP}]"
+
+distutils_enable_tests unittest
+distutils_enable_sphinx docs
+
+src_prepare() {
+	# Ships with incorrect mock import
+	sed -e 's/import mock/import unittest.mock as mock/' -i tests/test_cache.py || die
+
+	# Slew of tests that violate network-sandbox
+	sed -e 's/test_requests_from_cache/_test_requests_from_cache/' \
```

Comment:
Could you do some Bash magic like [here](https://gitweb.gentoo.org/repo/gentoo.git/tree/media-video/subliminal/subliminal-2.1.0-r1.ebuild#n84)? This is fine though.